src/OrchardCore.Modules/OrchardCore.Forms/wwwroot/Scripts/validationRuleHelper.js

```diff
@@ -0,0 +1,122 @@
+
+'use strict';
+var validationSet = new Set();
+var validationRuleApi = "/api/validationApi/ValidateFormByRule";
+
```

Comment:
Need to be tenant aware (e.g. including a tenant url prefix), and to support a possible virtual folder

E.g. by using the `Url` helper in a razor file and a `data-*` attribute whose value can be retrieved by the script.

Reply:
ok